setup.py

```diff
@@ -21,43 +23,43 @@ class build_ext(_build_ext):
     def finalize_options(self):
         super().finalize_options()
         import numpy
+
         self.include_dirs.append(numpy.get_include())
 
 
-cmdclass = {'build_ext': build_ext}
-ext = '.pyx' if USE_CYTHON else '.cpp'
+cmdclass = {"build_ext": build_ext}
+ext = ".pyx" if USE_CYTHON else ".cpp"
 extensions = [
     Extension(
-        name='zfits.' + name,
-        sources=[os.path.join('zfits', name + ext)],
-        extra_compile_args=['-std=c++0x'],
-        language='c++',
-        include_dirs=['zfits'],
+        name="zfits." + name,
+        sources=[os.path.join("zfits", name + ext)]
+        + [os.path.join("zfits", "remove_spikes_source.cpp")],
```

Comment:
this was needed .. since for some reason the file was not included into the sources before automatically.

Reply:
Minor style point, list addition is unnecessary, you can just have a single list literal